src/python/grpcio_status/grpc_status/rpc_status.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Reference implementation for status mapping in gRPC Python."""
+
+import grpc
+
+# Have to import google.protobuf before google.rpc, otherwise Python will fail
+#   to locate module rpc under google.
```

Comment:
https://github.com/bazelbuild/bazel/issues/6844

Reply:
The comment is updated with a link point to the unresolved issue and state the problem clearer.
```
# NOTE(https://github.com/bazelbuild/bazel/issues/6844)
# Due to Bazel issue, the namespace packages won't resolve correctly.
# Adding this unused-import as a workaround to avoid module-not-found error
# under Bazel builds.
```